src/bin/pg_autoctl/nodestate_utils.c

```diff
@@ -382,6 +391,92 @@ nodestateHealthToString(int health)
 }
 
 
+/*
+ * Transform the health column from a monitor into a single char.
+ */
+char
+nodestateHealthToChar(int health)
+{
+	switch (health)
+	{
+		case -1:
+		{
+			return '?';
+		}
+
+		case 0:
+		{
+			return '!';
```

Comment:
```suggestion
			return '✗';
```

Reply:
Same unicode remark with wider-than-space characters and table alignment.